package.json

```diff
@@ -27,6 +27,8 @@
     "async": "0.9.0",
     "cp-i18n-lib": "git+https://github.com/CoderDojo/cp-i18n-lib.git",
     "cp-logs-lib": "git://github.com/CoderDojo/cp-logs-lib#v1.0.1",
+    "cp-permissions-plugin": "git://github.com/CoderDojo/cp-permissions-plugin#0.0.1",
+    "crypto": "0.0.3",
```

Comment:
crypto is a part of node, this dependency is a 5 year old implementation no longer maintained. Should be removed.